node/server/arma_test.go

```diff
@@ -65,14 +63,17 @@ func TestCLI(t *testing.T) {
 }
 
 func TestLaunchArmaNode(t *testing.T) {
+	// TODO: remove all files and add a shut down signal to the CLI
```

Comment:
Lets do it in the next PR, it is actually pretty simple:

https://stackoverflow.com/questions/17620361/golang-equivalent-of-ansi-c-raise